templates/cisco_wlc_ssh_show_inventory.template

```diff
@@ -0,0 +1,13 @@
+Value BIA_MAC ([\da-zA-Z:]{17})
+Value MAX_AP_NUM (\d+)
+Value NAME (.*)
+Value DESCR (.*)
+Value PID ([\w\-]+)
+Value VID (\w+)
+Value SN (\w+)
+
+Start
+  ^Burned-in MAC Address\.+\s+${BIA_MAC}
```

Comment:
To prevent possible breakage of the template if the vendor changes spacing, can you use **\s*** or **\s+** instead of static spacing within the state?